devicon.json

```diff
@@ -11277,10 +11277,13 @@
         ],
         "versions": {
             "svg": [
-                "original"
+                "original",
+                "plain",
+                "line"
             ],
             "font": [
-                "original"
+                "plain",
+                "line"
             ]
         },
         "color": "#005fad",
```

Comment:
I'm unable to make a suggestion on the right line since it was not edited, but I believe an alias from line, original and plain versions can be added to their respective wordmark versions, since they fit both the wordmark and non-wordmark criteria.

Reply:
Sounds good. Added the elements as new aliases.